### PR TITLE
docs: added note about marking frontend config visible

### PR DESCRIPTION
**Pull request by Rugvip**
Common thing to stumble on, hoping this helps out a bit

### CODE REVIEW

File: docs/getting-started/contributors.md [hunk 1]
```diff
@@ -132,5 +132,10 @@ copy everything from the default config to the local config.
 `app-config.local.yaml` will be merged with `app-config.yaml` and overwrite the
 default app configs.
 
+> NOTE: If you want to add your own configuration values to access in the
+> frontend you also need to mark those values as visible using configuration
+> schema, either in the app or in your own plugin. For more information, see
+> [Defining Configuration](../conf/defining.md)
```
Comment (adamdmharvey) [inline docs/getting-started/contributors.md:138]:
```suggestion
> [Defining Configuration](../conf/defining.md).
```

**Review by adamdmharvey — APPROVED** [review]
👍

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4701-qyokcitg24li2bemnjxytlhunkezm1ji.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6037e0cffcb22d9da1607ba4

File: docs/getting-started/contributors.md [hunk 2]
```diff
@@ -132,5 +132,10 @@ copy everything from the default config to the local config.
 `app-config.local.yaml` will be merged with `app-config.yaml` and overwrite the
 default app configs.
 
+> NOTE: If you want to add your own configuration values to access in the
```
Comment (freben) [inline docs/getting-started/contributors.md:135]: Don't we have support for some kind of admonition / alert kinda thing for this type of block? Would be very nice to have note and warning at least, nicely styled

File: docs/getting-started/contributors.md (hunk 1 above)
Comment (freben) [inline docs/getting-started/contributors.md:138]:
```suggestion
> [Defining Configuration](../conf/defining.md).
```

**Review by freben — APPROVED** [review]

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4701-qyokcitg24li2bemnjxytlhunkezm1ji.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6037e0cffcb22d9da1607ba4

File: docs/getting-started/contributors.md (hunk 2 above)
Comment (Fox32) [reply docs/getting-started/contributors.md:135]: Docusaurus has something like that, but I wonder if that is already available <2.0.0 https://v2.docusaurus.io/docs/markdown-features#calloutsadmonitions

Comment (Fox32) [reply docs/getting-started/contributors.md:135]: image

I would interpret that as: no 😆

Comment (freben) [reply docs/getting-started/contributors.md:135]: Aight, but at least the `>` (as his code already uses, and as seen in your screenshot) stands out as well

**Pull request merged**